alertmanager/router.go

```diff
@@ -0,0 +1,47 @@
+package alertmanager
+
+import (
+	"sync"
+
+	"github.com/weaveworks/mesh"
+)
+
+// Router is the interface we use for a mesh router.
```

Comment:
I have no good intuition for how Go files within a package should be organised, so whatever you think is best.